src/ert/gui/model/snapshot.py

```diff
@@ -403,8 +405,10 @@ def _real_data(_index: QModelIndex, node: RealNode, role: int):
 
     @staticmethod
     def _job_data(
-        index: QModelIndex, node: ForwardModelStepNode, role: Qt.ItemDataRole
-    ):
+        index: QModelIndex,
+        node: ForwardModelStepNode,
+        role: int,  # Qt.ItemDataRole
```

Comment:
Same here^
Then we would not need the comment

Reply:
same as above